sdks/opik_optimizer/src/opik_optimizer/utils.py

```diff
@@ -126,61 +141,87 @@ def get_or_create_dataset(
     return dataset
 
 
-def _in_jupyter_environment() -> bool:
-    """
-    Check to see if code is running in a Jupyter environment,
-    including jupyter notebook, lab, or console.
-    """
+def _in_colab_environment() -> bool:
```

Comment:
All of these environment method are going to be removed in favor of ones in opik.